docs/core/whats-new/dotnet-8.md

```diff
@@ -11,21 +11,85 @@ author: gewarren
 
 .NET 8 is the successor to [.NET 7](dotnet-7.md). It will be [supported for three years](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a long-term support (LTS) release. You can [download .NET 8 here](https://dotnet.microsoft.com/download/dotnet).
 
-This article has been updated for .NET 8 Preview 6.
+This article has been updated for .NET 8 Preview 7.
 
 > [!IMPORTANT]
 >
 > - This information relates to a pre-release product that may be substantially modified before it's commercially released. Microsoft makes no warranties, express or implied, with respect to the information provided here.
 > - Much of the other .NET documentation on [https://learn.microsoft.com/dotnet](/dotnet) has not yet been updated for .NET 8.
 
-## Serialization
+## Core .NET libraries
+
+This section contains the following subtopics:
+
+- [Serialization](#serialization)
+- [Time abstraction](#time-abstraction)
+- [UTF8 improvements](#utf8-improvements)
+- [Methods for working with randomness](#methods-for-working-with-randomness)
+- [Performance-focused types](#performance-focused-types)
+- [System.Numerics and System.Runtime.Intrinsics](#systemnumerics-and-systemruntimeintrinsics)
+- [Data validation](#data-validation)
+- [Metrics](#metrics)
+- [Cryptography](#cryptography)
+- [Networking](#networking)
+- [Stream-based ZipFile methods](#stream-based-zipfile-methods)
+
+### Serialization
 
 Many improvements have been made to <xref:System.Text.Json?displayProperty=fullName> serialization and deserialization functionality including:
 
+- The serializer has built-in support for the following additional types. Previously, code that attempted to serialize these types threw an <xref:System.InvalidOperationException> at run time.
```

Comment:
```suggestion
- The serializer has built-in support for the following additional types.
```
That's not strictly accurate. In many cases unsupported types would simply get serialized as empty objects `{}` since STJ will attempt to serialize all types, even those it doesn't recognize.